packages/@react-spectrum/breadcrumbs/src/Breadcrumbs.tsx

```diff
@@ -211,7 +212,7 @@ function Breadcrumbs<T>(props: SpectrumBreadcrumbsProps<T>, ref: DOMRef) {
 
     return (
       <li
-        key={key}
+        key={isCurrent ? 'current' : key}
```

Comment:
The above works if we make the current item always have a consistent React key so that it always has the same DOM node. That way we can focus the current item even before a re-render occurs, and it will still be the right DOM node afterward.

Reply:
I've tested this and it does not work. Focus gets lost on action when the breadcrumbs re-render to display a new location.